### PR TITLE
Update to HeroIcons v2.1.5

### PR DESCRIPTION
**Pull request by goncalotomas**
A few new icons have been added since `2.1.1`. This PR updates the project dependencies and moves the current HeroIcons version to `2.1.5`. Open to feedback of course :)

### CODE REVIEW

File: assets/heroicons.exs [hunk 1]
```diff
@@ -112,7 +112,7 @@ defmodule Heroicons do
     """
   end
 
-  <%= for  {func, [outline, solid, mini, micro]} = icon when not is_nil(micro) <- @icons do %>
+  <%= for  {func, [outline, solid, mini, micro]} = _icon when not is_nil(micro) <- @icons do %>
```
Comment (goncalotomas) [inline assets/heroicons.exs:115]: both changes in this file fix a compile warning

**Comment by greven** [comment]
Was about to submit a PR for this, I know that the default in Phoenix now is to use Tailwind and the icons with CSS, but as someone that don't use Tailwind in my current project, this package is still very useful. :)

@mveytsman can we get this merged, please?

**Review by mveytsman — APPROVED** [review]

**Comment by mveytsman** [comment]
@goncalotomas thank you!

Do you mind fixing the failing tests?

**Comment by goncalotomas** [comment]
These test failures are pretty interesting in that they relate to the order of attributes after being rendered out, e.g. it seems that in OTP versions before 26 the svg will render out to:

```
<svg xmlns=\"http://www.w3.org/2000/svg\" aria-hidden=\"true\" fill=\"none\" stroke=\"currentColor\" stroke-width=\"1.5\" viewBox=\"0 0 24 24\">
[...]
</svg>
```
and after OTP26 it renders out to this:
```
"<svg xmlns=\"http://www.w3.org/2000/svg\" aria-hidden=\"true\" fill=\"none\" viewBox=\"0 0 24 24\" stroke=\"currentColor\" stroke-width=\"1.5\">
[...]
</svg>"
```

The main difference being that the `class` and `viewBox` attributes get listed in a different order, and I suspect it's related to map ordering changes in OTP26 / Elixir 1.14 [(more info)](https://fly.io/phoenix-files/taking-control-of-map-sort-order-in-elixir/). I was able to replicate the error by going back to OTP24.2 and Elixir 1.14.5: using the same versions as CI made the tests fail, but switching to OTP26 and Elixir 1.16 these tests pass.

**TL;DR** If we make the tests work for current OTP/Elixir, they break in the older versions and vice versa.

There's a decision to make, and I see three ways forward:
- We stay aware of this for further changes to the library but decide to just keep things the way they are for now
- We improve the tests so that they don't particularly care about the ordering of attributes
- We patch the tests so they work with current OTP/Elixir and stop testing for < OTP26

All of these have different trade-offs, so I'll leave it up to @mveytsman to decide :)
Also, we don't have to decide now. I've pushed a commit that solves the issue in CI, but any future updates will most likely run into this issue again.

Have a great day!

**Comment by mveytsman** [comment]
@goncalotomas thank you so much for figuring this out and writing it up!

I think the right move is to keep this in mind and fix it next release, I'll see about improving the tests!

**Comment by mveytsman** [comment]
Published!
